modules/plugin/src/main/scala/com/siriusxm/snapshot/Snapshot4sPlugin.scala

```diff
@@ -44,9 +43,9 @@ object Snapshot4sPlugin extends AutoPlugin {
 /** This object was generated by sbt-snapshot4s. */
 object generated {
   implicit val snapshotConfig: SnapshotConfig = new SnapshotConfig(
-    resourceDirectory = Path("${snapshot4sResourceDirectory.value}"),
-    outputDirectory = Path("${snapshot4sDirectory.value}"),
-    sourceDirectory = Path("${sourceBaseDirectory((Test / sourceDirectories).value)}")
+    resourceDirectory = Path("${snapshot4sResourceDirectory.value.getAbsolutePath.replace("\\","/")}"),
```

Comment:
The compile error is caused by the fact that the backslash character `\` isn't escaped.

Instead of changing the character, it would be safer to escape the character by using a triple-quoted string. This would escape any other special characters in filepaths.

Could you try doing this with a `tripleQuote` variable, for example?

```scala
val tripleQuote = "\"\"\""

val text = s"""
...
    resourceDirectory = Path($tripleQuote${snapshot4sResourceDirectory.value}$tripleQuote),
...
```